core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala

```diff
@@ -84,6 +84,7 @@ private[ui] class ExecutorTable(stageId: Int, stageAttemptId: Int, parent: Stage
           <th>Shuffle Spill (Memory)</th>
           <th>Shuffle Spill (Disk)</th>
         }}
+        <th>Logs</th>
```

Comment:
Should we possibly call this Executor Logs instead? It could clarify that the log isn't just for this stage but the whole log for the executor it was run on.